src/actions.js

```diff
@@ -244,11 +244,6 @@ const handleFetchUser = ({ status, data }) => {
       type: 'RECEIVE_USER',
       user: data,
     };
-  case 401:
```

Comment:
i dont think that removing this will hurt anything, it was added in https://github.com/uc-cdis/data-portal/pull/74

Reply:
yes it happens when user is not logged in but it looks to be not needed